src/classes/RD2_ElevateIntegrationService.cls

```diff
@@ -34,6 +34,15 @@
 * @description Services related to the RecurringDonation <-> Elevate integration
 */
 public inherited sharing class RD2_ElevateIntegrationService {
+
+    /***
+     * @description Contains a prefix appended to a GUID to indicate the RD is in draft
+     * and has a temp commitment Id so future installment Opportunities are not
+     * created for this Recurring Donation and any user can change it
+     * from a temp value to the actual value.
+     */
+    public static final String TEMP_COMMITMENT_ID_PREFIX = '_TEMP_';
```

Comment:
What would you think of `_PENDING_` for this instead of `_TEMP`?

Reply:
Perfect, will make the change now.